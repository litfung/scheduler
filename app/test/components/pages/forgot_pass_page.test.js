import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';

import ForgotPasswordPage from '../../../src/client/components/pages/forgot_pass_page';
import configureStore from '../../../src/helpers/configure_store';

test('Snapshot for ForgotPasswordPage Component', () => {
  const wrapper = shallow(
    <Provider store={configureStore({ get: jest.fn(() => {}) })}>
      <ForgotPasswordPage.component
        handleAppMode={jest.fn(() =>  {})}
      />
    </Provider>
  );
  expect(wrapper).toMatchSnapshot();
});
