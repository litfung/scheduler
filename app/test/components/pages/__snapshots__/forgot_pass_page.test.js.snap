// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot for LoginPage Component 1`] = `
<Connect(ForgotPasswordPage)
  handleAppMode={[MockFunction]}
/>
`;
